threedi_modelchecker/tests/test_error_messages.py

```diff
@@ -0,0 +1,24 @@
+from threedi_schema.domain.models import DECLARED_MODELS
+
+from threedi_modelchecker.config import Config
+
+checks = Config(models=DECLARED_MODELS).checks
+
+
+def test_error_description():
```

Comment:
Name implies that this only tests error. But I guess/hope this also includes warnings.

Reply:
true, perhaps test_check_descriptions would be a better name